### DISCUSSION TITLE
Object detection confused by other movement

### DISCUSSION THREAD

**Discussion by HSB-collab** [blakeblackshear/frigate#1660] [Ask A Question]
Hi I have a camera setup to detect cars. It does that well but unfortunately I get a high number of false detections because I have a car parked in the line of site of the camera. If there is other movement (e.g. trees moving) or a change in the light in the frame of the camera it then detects my parked car and gives a notification. 

I attach the snapshot and clip from one of these false detections. 

I realise I could put a mask in but that would be difficult because of the shape of my driveway and also I don't think it would block out the change in the light.

Is there anything I can do about this?
![snapshot](https://user-images.githubusercontent.com/72898374/131445670-19b59be2-b13e-4ebf-a751-4af33776f1c6.jpg)

https://user-images.githubusercontent.com/72898374/131445675-1b2b1ba1-edc1-4a93-ba58-b886388f4093.mp4

**Comment by Pascal66** [comment]
Same for me since 0.9 RC1 & RC2. Cant put a mask or filter by zone
Car is detected as person (10 times per day), and even if a person is here, the the parked car is prefered
Not happen on 0.8
![snapshot-1630390713 76858-s7miw7](https://user-images.githubusercontent.com/3041616/131453221-a6df35f5-8dcd-4371-b82e-b25150ff4f71.jpg)

**Reply by HSB-collab** [reply]
I am still on 0.8

**Reply by Pascal66** [reply]
Lucky. But same problem

**Comment by blackgold9** [comment]
Came here to post about this. What I think I'm looking for here is the ability to know if an object is "New".

**Comment by s24dor** [comment]
I would mask out everything except your driveway and area Infront of garage, but definitely mask the road area. If you are only interested in cars coming and going then also mask out the area in garage where the cars park.
